server/index.js

```diff
@@ -1,26 +1,31 @@
 var express = require('express');
 var bodyParser = require('body-parser');
-var { travelData } = require('../database/Travel.js');
+var { getTravelData } = require('../database/Travel.js');
 
 var app = express();
 var PORT = 3001;
 
 app.use(bodyParser.json());
 app.use(express.static(__dirname + '/../client/dist'));
 
-app.get('/attraction_product_detail', (req, res) => {
-  travelData()
-  .then(data => res.send(data))
-  .catch(() => res.sendStatus(500));
-});
-
-
-
-
-
-
+app.get('/api/events', (req, res) => {
+  res.send("Route Connected");
+  
+  getTravelData()
+  .then((data) => {
+    // console.log(data);
+  });
```

Comment:
`app.get('/api/events', (req, res) => {
  res.send("Route Connected");

  getTravelData()
  .then((data) => {
    // console.log(data);
  });`

Reply:
I am not really sure what you are trying to do here? Can you walk me through it?

Reply:
Here I'm setting a response from the success value returned by getTravelData which is a method to access database for get all data.